spirv_msl.cpp

```diff
@@ -1004,9 +1052,11 @@ void CompilerMSL::emit_fixup()
 	{
 		if (options.vertex.fixup_clipspace)
 		{
-			const char *suffix = backend.float_literal_suffix ? "f" : "";
+			/*const char *suffix = backend.float_literal_suffix ? "f" : "";
 			statement(qual_pos_var_name, ".z = 2.0", suffix, " * ", qual_pos_var_name, ".z - ", qual_pos_var_name,
-			          ".w;", "    // Adjust clip-space for Metal");
+			          ".w;", "    // Adjust clip-space for Metal");*/
+			statement(qual_pos_var_name, ".z = (", qual_pos_var_name, ".z + ", qual_pos_var_name,
```

Comment:
If I read it right, this compacts [-1, 1] range to [0, 1] instead of the opposite, any particular reason this change is done?